hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationRunnerImpl.java

```diff
@@ -272,7 +272,11 @@ private boolean run(Operation op, long startNanos) {
                 if (op instanceof PartitionIteratingOperation) {
                     c = ((PartitionIteratingOperation) op).getOperationFactory().getClass();
                 }
-                LatencyDistribution distribution = opLatencyDistributions.computeIfAbsent(c, k -> new LatencyDistribution());
+                LatencyDistribution distribution = opLatencyDistributions.get(c);
+                // Note: we want to prevent lock here, if collision happened.
```

Comment:
this comment applies to the line above as well, right?
(so try get and only if this returns null, use computeIfAbsent)

Reply:
Yes